docs/tutorial/tray.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Tray
+description: This guide will take you through the process of creating a Tray icon with its own context menu to the system's notification area.
+slug: tray
+hide_title: true
+---
+
+# Tray
+
+## Overview
+
+<!-- ✍ Update this section if you want to provide more details -->
+
+This guide will take you through the process of creating a [Tray](https://www.electronjs.org/docs/api/tray) icon with its own context menu to the system's notification area.
+
+On MacOS and Ubuntu, the Tray will be located on the top right corner of your screen, adjacent to your battery and wifi icons.  On Windows, the Tray will be located in the bottom right corner.
```

Comment:
Not sure how useful this detail is given that the position can be changed on Windows. Not sure what the best way around this would be.